viewing_party/party.py

```diff
@@ -1,23 +1,126 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dictionary = {}
+    if title and genre and rating:
+        movie_dictionary["title"] = title
+        movie_dictionary["genre"] = genre
+        movie_dictionary["rating"] = rating
+    else:
+        return None
+    
+    return movie_dictionary
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+
+    return user_data
+
+def watch_movie(user_data, title):
+    copy_user_data = user_data

```

Comment:
Recall that when we assign a variable to another variable then the 2 will point to the same list in memory. From the README, we expect that this method will have a side effect that will directly modify user_data so we don't need to make a copy or reassign values to try to get around a side effect.